docs/admin/logging/overview.md

```diff
@@ -130,6 +131,10 @@ By default, Docker uses the first 12 characters of the container ID to tag log m
 Refer to the [log tag option documentation](log_tags.md) for customizing
 the log tag format.
 
+`syslog-format` configures syslog protocol version to use when logging,
+Allowed value is rfc5424 in order to use RFC5424 syslog protocol. 
+Default is RFC3164 incompatible syslog protocol.
```

Comment:
@thaJeztah  Here are the comments from srslog implementation:
-> DefaultFormatter is the original format supported by the Go syslog package,
// and is a non-compliant amalgamation of 3164 and 5424 that is intended to
// maximize compatibility.
-> UnixFormatter omits the hostname, because it is only used locally.
-> RFC3164Formatter provides an RFC 3164 compliant message.
-> RFC5424Formatter provides an RFC 5424 compliant message.

Note that original srslog package (before changes I took)
https://github.com/RackSec/srslog/blob/68fa06c8afc0374f78d4be6107081b17a2c349c9/srslog_unix.go#L40, actually used something that is similar to latest version Unixformatter, 
See the quotation below:

> 
writeString formats syslog messages using time.Stamp instead of time.RFC3339,
// and omits the hostname (because it is expected to be used locally).
>